apps/src/blockTooltips/DropletAutocompleteParameterTooltipManager.js

```diff
@@ -65,8 +68,63 @@ DropletAutocompleteParameterTooltipManager.prototype.onCursorMovement_ = functio
     return;
   }
 
+  if (this.startingAutoComplete) {
+    // Guard against re-entrancy that occurs inside the showParamDropdownIfNeeded_() below
```

Comment:
Ah I see, the `startAutocomplete` command in turn ends up calling the `changeCursor` command?